### ISSUE TITLE
Missing libxml2 for MentaLiST in conda

### ISSUE THREAD

**Issue by apetkau** [WGS-TB/MentaLiST#14]
Hey,

I've been using MentaLiST on my desktop and it's working great.  But, on our cluster I get:

```
$ mentalist list_pubmlst
signal (11): Segmentation fault
while loading /share/apps/miniconda2/envs/mentalist@0.2.0/bin/mentalist, in expression starting on line 271
unknown function (ip: 0x3f13f2916f)
__xmlParserInputBufferCreateFilename at /usr/lib64/libxml2.so (unknown line)
xmlNewInputFromFile at /usr/lib64/libxml2.so (unknown line)
xmlCreateURLParserCtxt at /usr/lib64/libxml2.so (unknown line)
xmlSAXParseFileWithData at /usr/lib64/libxml2.so (unknown line)
parse_file at /share/apps/miniconda2/envs/mentalist@0.2.0/share/julia/site/v0.5/LightXML/src/document.jl:88
_pubmlst_xml at /share/apps/miniconda2/envs/mentalist@0.2.0/bin/mlst_download_functions.jl:22
...
```

I'm thinking that our cluster has an out of date version of `libxml2`.  I am able to fix this using conda by running:

```
conda install libxml2
```

I'm wondering if `libxml2` can be added to the bioconda dependencies for MentaLiST?

Thanks,

Aaron

**Comment by dfornika** [comment]
Thanks for letting us know. We'll create new builds that include `libxml2`.

**Comment by dfornika** [comment]
This has been fixed in the v0.2 conda package (build `mentalist-0.2.0-1`). I'll submit a pull request to fix it in v0.1 soon.

**Comment by apetkau** [comment]
Awesome, thanks :+1:
